### PR TITLE
chore: Rename Outputs to Actions

### PR DESCRIPTION
**Pull request by simar7**
This simplifies the naming convention by replacing all Outputs as Actions.

Signed-off-by: Simar <simar@linux.com>

### CODE REVIEW

**Review by krol3 — COMMENTED** [review]
LGTM @simar7 and works too , what about the postee-ui? will we need to change the labels too?

**Comment by simar7** [comment]
> LGTM @simar7 and works too , what about the postee-ui? will we need to change the labels too?

Did I miss any? I changed them in the UI only, not the code for backend/frontend modules.

File: ui/backend/uiserver/server.go [hunk 1]
```diff
@@ -50,7 +50,7 @@ func Instance(webLocalPath, port, cfg, webhookUrl, admusr string, admpwd string)
 	server.router.HandleFunc("/api/config", server.updateConfig).Methods("POST")
 	server.router.HandleFunc("/api/config", server.getConfig).Methods("GET")
 	server.router.HandleFunc("/api/test", server.testSettings).Methods("POST")
-	server.router.HandleFunc("/api/outputs/stats", server.plgnStats).Methods("GET")
+	server.router.HandleFunc("/api/actions/stats", server.plgnStats).Methods("GET")
```
Comment (AndreyLevchenko) [inline ui/backend/uiserver/server.go:53]: I didn't find corresponding changes in frontend

**Review by AndreyLevchenko — CHANGES_REQUESTED** [review]

File: ui/backend/uiserver/server.go (hunk 1 above)
Comment (simar7) [reply ui/backend/uiserver/server.go:53]: Good catch - this was not supposed to be changed.

**Pull request merged**